preinstall.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mv semantic.json.example semantic.json
```

Comment:
This could be `cp` instead of moving.

Reply:
It could be yes, actually it doesn't really matter, since this preinstall will be launched only by Travis in a VM. But if you want, I can modify `mv` to `cp`.

Reply:
I think the Travis installation should be identical to how it is installed for users therefore it doesn't lead to any problems.

Reply:
No problem, will do then 😃

Reply:
Done in 76027c6.